### DISCUSSION TITLE
Display popup text using location

### DISCUSSION THREAD

**Discussion by alex27riva** [CariusLars/ar_flutter_plugin#141] [Q&A]
Hi everyone, I'm working on a mobile app to enhance the territory of a small municipality, I would like to show through augmented reality popups to indicate some points of interest.
All the POIs have been mapped using GPS.

Something like this:

![AR popup example](https://user-images.githubusercontent.com/5512235/191771656-00eb1cf3-5753-4498-9904-c6f76bead80b.png)

Is it possible through this plugin?

Thanks.

Bye

**Comment by leeprobert** [comment]
It may be possible with Cloud Anchors but I would also look into using Google Maps or MapBox SDK to do this. They both have 3D orientations that can be mapped over your camera view and use location, gps, gyro to achieve something like this.
